src/GraphQL/Types/QueryArgument.cs

```diff
@@ -83,7 +83,7 @@ public string Name
         /// <summary>
         /// Gets or sets the default value of the argument.
         /// </summary>
-        public object? DefaultValue { get; set; }
+        public object? DefaultValue { get; set; } // TODO: [NotAGraphType]?
```

Comment:
This property contains an actual value (such as "abc"), not a type, so probably not necessary.